app/docs/introduction/featured-posts.tsx

```diff
@@ -42,6 +42,7 @@ export default async function FeaturedPosts() {
                       alt={post.altText}
                       width={imageWidth}
                       height={imageHeight}
+                      style={{ height: `${imageHeight}px` }}
```

Comment:
If you want to standardize style properties like this you want to describe a class! spin up a new class by creating a .css file within the introduction directory, or if you want to reuse it more places lift it higher up. The highest level is global.css inside /app

Reply:
> a

I don't yet have a need to standardize it, it was just a quick fix to force override the height-auto that was being applied from somewhere. I just wanted the featured posts to be the same height regardless of post order, otherwise since it had 2 col on larger screens, it was matching the height of one with the other beside it causing improper post sizes.